lib/eclipse/share/keywords/001_Eclipse300/W/WATDENT

```diff
@@ -10,8 +10,8 @@
   "comment": "item, the RM is _very_ inconsistent. (it says that the default for the second item is 1.67/degR in",
   "comment": "FIELD units which is the same the 3e-4/K for METRIC but four orders of magnitude off!)",
   "items" : [
-    {"name" : "REFERENCE_TEMPERATURE" , "value_type" : "DOUBLE", "dimension" : "AbsoluteTemperature", "default": 527.67 },
-    {"name" : "EXPANSION_COEFF_LINEAR" , "value_type" : "DOUBLE", "dimension" : "1/AbsoluteTemperature", "default": 1.67e-4 },
-    {"name" : "EXPANSION_COEFF_QUADRATIC" , "value_type" : "DOUBLE", "dimension" : "1/AbsoluteTemperature*AbsoluteTemperature", "default": 9.26e-7 }
+    {"name" : "REFERENCE_TEMPERATURE" , "value_type" : "DOUBLE", "dimension" : "AbsoluteTemperature", "default": 293.15 },
```

Comment:
Holy diver - do we have two *different* default temperatures?

Reply:
at least if the documentation and what I've been told about it is correct.